### PR TITLE
email new applicant acknowledging their application was received

### PR DESCRIPTION
**Pull request by weedySeaDragon**
### PT Story:  Send applicant email acknowledging that their application was received
https://www.pivotaltracker.com/story/show/151614904

## Changes proposed in this pull request:
1.  Send email when a new membership application is successfully submittted (saved)
2. In the flash message that appears, tell them that email is on the way.

Ready for review:
@AgileVentures/shf-project-team

### CODE REVIEW

File: config/locales/sv.yml [hunk 1]
```diff
@@ -263,7 +263,7 @@ sv:
       submit_button_label: *submit
 
     create:
-      success: Tack, din ansökan har skickats.
+      success: "Tack, din ansökan har skickats. Du borde få ett mail inom kort på %{email_address} med information om vad du kan förvänta sig nästa."
```
Comment (thesuss) [inline config/locales/sv.yml]: success: "Tack, din ansökan har skickats. Du borde få en bekräftelse inom kort till %{email_address}."

**Review by thesuss — CHANGES_REQUESTED** [review]

**Review by thesuss — APPROVED** [review]

File: features/emails/new-applicantion-gets-ack-email.feature [hunk 2]
```diff
@@ -0,0 +1,58 @@
+Feature: New Applicant gets an email acknowledging their application
+
+  As a new applicant,
+  So that I know that SHF received my application and that I didn't do something wrong,
+  and explaining what I can expect to happen next,
+  I should get an email acknowleding my new application
```
Comment (patmbolger) [inline features/emails/new-applicantion-gets-ack-email.feature]: ^^ mispelling

Comment (patmbolger) [reply features/emails/new-applicantion-gets-ack-email.feature]: Also, please fix the misspelling in the file name.

Comment (patmbolger) [reply features/emails/new-applicantion-gets-ack-email.feature]: Actually, this file looks like it was replaced by the one below, so it should be removed(?)

File: features/emails/new-application-gets-ack-email.feature [hunk 3]
```diff
@@ -0,0 +1,48 @@
+Feature: New Applicant gets an email acknowledging their application
+
+  As a new applicant,
+  So that I know that SHF received my application and that I didn't do something wrong,
+  and so I know what I should expect to happen next,
+  I should get an email acknowleding my new application
```
Comment (patmbolger) [inline features/emails/new-application-gets-ack-email.feature]: mispelling

**Review by patmbolger — APPROVED** [review]

**Review by RobertCram — APPROVED** [review]

**Review by RobertCram — APPROVED** [review]
Thanks for leaving this open. I learned a lot "reviewing" it.

**Pull request merged**